pyvista/plotting/renderer.py

```diff
@@ -2606,25 +2606,49 @@ def set_background(self, color, top=None):
             self.GradientBackgroundOff()
         self.Modified()
 
-    def set_environment_texture(self, texture):
+    def set_environment_texture(self, texture, is_srgb=False):
         """Set the environment texture used for image based lighting.
 
         This texture is supposed to represent the scene background. If
         it is not a cubemap, the texture is supposed to represent an
         equirectangular projection. If used with raytracing backends,
         the texture must be an equirectangular projection and must be
-        constructed with a valid vtkImageData. Warning, this texture
-        must be expressed in linear color space. If the texture is in
-        sRGB color space, set the color flag on the texture or set the
-        argument isSRGB to true.
+        constructed with a valid ``vtk.vtkImageData``.
 
         Parameters
         ----------
         texture : vtk.vtkTexture
             Texture.
+
+        is_srgb : bool, optional
+            If the texture is in sRGB color space, set the color flag on the
+            texture or set this parameter to ``True``. Textures are assumed
+            to be in linear color space by default.
+
+        Examples
+        --------
+        Add a skybox cubemap as an environment texture and show that the
+        lighting from the texture is mapped on to a sphere dataset. Note how
+        even when disabling the default lightkit, the scene lighting will still
+        be mapped onto the actor.
+
+        >>> from pyvista import examples
+        >>> import pyvista as pv
+        >>> pl = pv.Plotter(lighting=None)
+        >>> cubemap = examples.download_sky_box_cube_map()
+        >>> _ = pl.add_mesh(pv.Sphere(), pbr=True, metallic=0.9, roughness=0.4)
+        >>> pl.set_environment_texture(cubemap)
+        >>> pl.camera_position = 'xy'
+        >>> pl.show()
+
         """
+        # cube_map textures cannot use sphrical harmonics
+        if texture.cube_map:
+            self.AutomaticLightCreationOff()
+            self.UseSphericalHarmonicsOff()
```

Comment:
```suggestion
            # disable spherical harmonics was added in 9.1.0
            if hasattr(self, 'UseSphericalHarmonicsOff'):
                self.UseSphericalHarmonicsOff()
```

Reply:
Derp, we can then add back the new test for 9.0.3. Sorry.